pkg/storage/metastore/kv.go

```diff
@@ -15,15 +15,29 @@ package metastore
 
 import (
 	"encoding/binary"
+	"fmt"
 
 	pb "github.com/parca-dev/parca/gen/proto/go/parca/metastore/v1alpha1"
 )
 
-//const stacktraceKeyPrefix = "stacktrace/by-key/"
+const (
+	v1prefix = "v1"
+)
 
-const stacktraceIDPrefix = "stacktrace/by-id/"
+const (
+	//const stacktraceKey = "stacktrace/by-key/"
+	stacktraceID = "stacktrace/by-id/"
+	locationsKey = "locations/by-key/"
+	functionKey  = "functions/by-key/"
+	mappingKey   = "mappings/by-key/"
```

Comment:
Since we only have one version we don't need to compute these really. It's mostly a matter of figuring out the layout of the key:

Should we either pick
```suggestion
	stacktraceID = "stacktrace/by-id/v1"
	locationsKey = "locations/by-key/v1"
	functionKey  = "functions/by-key/v1"
	mappingKey   = "mappings/by-key/v1"
```
or
```suggestion
	stacktraceID = "v1/stacktrace/by-id/"
	locationsKey = "v1/locations/by-key/"
	functionKey  = "v1/functions/by-key/"
	mappingKey   = "v1/mappings/by-key/"
```

What do you think?

Reply:
I prefer the second option